src/app/home/shared/forms/budgets-form/budgets-form.component.ts

```diff
@@ -64,8 +66,16 @@ export class BudgetsFormComponent implements OnInit {
       this.budget.repeatCycle = this.repeatCycle;
       this.budget.uid = this.authService.currentUser.uid;
       this.apiService.addBudget(this.budget);
-
-      this.isNewBudgetCardVisible = false;
+      
+      this.closeNewBudgetCardEmitter.emit();
```

Comment:
I can see a function below that does this. Consider using that function here to maintain single source of truth.